src/domain/occurrences/__tests__/OccurrencesPage.test.tsx

```diff
@@ -0,0 +1,122 @@
+import { MockedResponse } from '@apollo/react-testing';
+import { format } from 'date-fns';
+import { advanceTo, clear } from 'jest-date-mock';
+import React from 'react';
+
+import { EventDocument, OccurrenceNode } from '../../../generated/graphql';
+import {
+  fakeEvent,
+  fakeLocalizedObject,
+  fakeOccurrences,
+  fakePEvent,
+} from '../../../utils/mockDataUtils';
+import {
+  renderWithRoute,
+  screen,
+  userEvent,
+  waitFor,
+} from '../../../utils/testUtils';
+import { ROUTES } from '../../app/routes/constants';
+import OccurrencesPage from '../OccurrencesPage';
+
+const eventName = 'Testitapahtuma';
+const eventMock = fakeEvent({
+  name: fakeLocalizedObject(eventName),
+  startTime: '2020-07-13T05:51:05.761000Z',
+});
+
+const fakeOccurrenceOverrides: Partial<OccurrenceNode>[] = [
+  {
+    startTime: '2020-08-03T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-04T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-05T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-06T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-07T09:00:00+00:00',
+  },
+];
+
+const apolloMocks: MockedResponse[] = [
+  {
+    request: {
+      query: EventDocument,
+      variables: {
+        id: eventMock.id,
+        include: ['location'],
+      },
+    },
+    result: {
+      data: {
+        event: {
+          ...eventMock,
+          pEvent: fakePEvent({
+            occurrences: fakeOccurrences(5, fakeOccurrenceOverrides),
```

Comment:
Could write it also:
`occurrences: fakeOccurrences(fakeOccurrenceOverrides.length, fakeOccurrenceOverrides),`